docs/index.md

```diff
@@ -4,9 +4,10 @@ title: Welcome to Webaverse
 hide_title: true
 ---
 
+<div style='margin-top: -60px'></div>
 <h1 align="center"><b>🌌 Welcome To A Universe Of Augmentations ⚙️</b></h1>
 
-<img alt="Hands Reality Tab" src="https://user-images.githubusercontent.com/29695350/55507781-0e463300-561e-11e9-9b1a-f43b8259d041.gif">
+<img alt="Hands Reality Tab" src="https://user-images.githubusercontent.com/29695350/55507781-0e463300-561e-11e9-9b1a-f43b8259d041.gif" width="60%">
```

Comment:
I think this image is a little distracting -- it is actually a placeholder that isn't from this project at all.

We should either have screencaptures of things like is done in readme of https://github.com/aframevr/aframe, or maybe an infographic, or nothing at all for now.

Reply:
Resolved in https://github.com/webaverse/docs/pull/17